yaptide/batch/batch_methods.py

```diff
@@ -327,3 +327,31 @@ def delete_job(simulation: BatchSimulationModel, user: KeycloakUserModel,
         return {"message": "Job cancelation failed"}, 500
 
     return {"message": "Job canceled"}, 200
+
+
+def cancel_simulation_with_fetching_data(simulation: BatchSimulationModel, user: KeycloakUserModel,
+                                         cluster: ClusterModel) -> tuple[dict, int]:  # skipcq: PYL-W0613
```

Comment:
```suggestion
                                         cluster: ClusterModel) -> tuple[dict, int]:  # skipcq: PYL-W0613
```

why do you need `# skipcq: PYL-W0613` ?

Reply:
I copied the above function, I didn't know it had any use (in fact I didn't even think about it)

Reply:
The please get rid of `# skipcq: PYL-W0613` and see if deepsource complains